Jenkinsfile

```diff
@@ -53,10 +63,17 @@ library(
 List arches = params.ARCHES.tokenize(',')
 def errorMessages = ''
 def config = MAQEAPI.v1.getProvisioningConfig(this)
+config.jobgroup = 'multiarch-qe'
 
-MAQEAPI.v1.runParallelMultiArchTest(
+def targetHost = MAQEAPI.vi.newTargetHost()
+targetHost.hostname = params.PREPROVISIONED_HOST
+targetHost.name = params.PREPROVISIONED_HOST
+targetHost.arch = 'x86_64'
+targetHost.provisioner = 'NOOP'
+
```

Comment:
Add your old withCredentials block here to set the scriptParams with the OSP credentials so we can complete the git clone :)